inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sort/SortServiceImplTest.java

```diff
@@ -195,20 +216,50 @@ private void prepareDataNode(String taskName) {
     }
 
     private void prepareGroupId(String groupId) {
-        InlongGroupEntity entity = new InlongGroupEntity();
-        entity.setInlongGroupId(groupId);
-        entity.setInlongClusterTag(TEST_TAG);
-        entity.setMqResource(TEST_TOPIC);
-        entity.setMqType("PULSAR");
-        entity.setName("testName");
-        entity.setDescription("testDescription");
-        entity.setCreator(TEST_CREATOR);
-        entity.setInCharges(TEST_CREATOR);
-        entity.setCreateTime(new Date());
-        entity.setModifyTime(new Date());
-        entity.setIsDeleted(InlongConstants.UN_DELETED);
-        entity.setVersion(InlongConstants.INITIAL_VERSION);
-        inlongGroupEntityMapper.insert(entity);
+        InlongPulsarRequest request = new InlongPulsarRequest();
+        request.setInlongGroupId(groupId);
+        request.setMqResource("test namespace");
+        request.setInlongClusterTag(TEST_TAG);
+        request.setVersion(InlongConstants.INITIAL_VERSION);
+        request.setName("test group name");
```

Comment:
Suggest adding `-` or `_` between words for name, backup topic, etc.

Reply:
fixed, thx